server/views/pages/dashboard.njk

```diff
@@ -44,88 +46,92 @@
-        maxDate: todayAsShortDate,
-        errorMessage: errors | findFieldInGovukErrorSummary("toDate"),
-        leadingZeros: "true"
-      }) }}
+      <div class="govuk-grid-row">
+
+        {% if showEstablishmentsFilter %}
+          <div class="govuk-grid-column-one-half-from-desktop">
+            {{ accessibleAutocomplete({
+              id: "location",
+              name: "location",
+              label: {text: "Establishment"},
+              items: establishments | govukSelectInsertDefault("All establishments") | govukSelectSetSelected(formValues.location),
+              value: formValues.location or "",
+              errorMessage: errors | findFieldInGovukErrorSummary("location"),
+              class: "govuk-!-width-half",
+              nonce: cspNonce
+            }) }}
+          </div>
+        {% endif %}
 
-      {% if showEstablishmentsFilter %}
-        {{ accessibleAutocomplete({
-          id: "location",
-          name: "location",
-          label: {text: "Establishment"},
-          items: establishments | govukSelectInsertDefault("All establishments") | govukSelectSetSelected(formValues.location),
-          value: formValues.location or "",
-          errorMessage: errors | findFieldInGovukErrorSummary("location"),
-          class: "govuk-!-width-half",
-          nonce: cspNonce
-        }) }}
-      {% endif %}
+        <div class="govuk-grid-column-one-half-from-desktop">
+          {{ accessibleAutocomplete({
+            id: "incidentType",
+            name: "incidentType",
+            label: {text: "Incident type"},
+            items: incidentTypes | govukSelectInsertDefault("All incident types") | govukSelectSetSelected(formValues.incidentType),
+            value: formValues.incidentType or "",
+            errorMessage: errors | findFieldInGovukErrorSummary("incidentType"),
+            class: "govuk-!-width-half",
+            nonce: cspNonce
+          }) }}
+        </div>
 
-      {{ accessibleAutocomplete({
-        id: "incidentType",
-        name: "incidentType",
-        label: {text: "Incident type"},
-        items: incidentTypes | govukSelectInsertDefault("All incident types") | govukSelectSetSelected(formValues.incidentType),
-        value: formValues.incidentType or "",
-        errorMessage: errors | findFieldInGovukErrorSummary("incidentType"),
-        class: "govuk-!-width-half",
-        nonce: cspNonce
-      }) }}
+      </div>
 
-      {{ govukCheckboxes({
-        classes: "govuk-checkboxes--small",
-        name: "incidentStatuses",
-        id: "incidentStatuses",
-        fieldset: {
-          legend: {
-            text: statusCheckboxLabel,
-            classes: "govuk-fieldset__legend--s"
-          }
-        },
-        items: statusItems | govukMultipleCheckedItems(formValues.incidentStatuses),
-        errorMessage: errors | findFieldInGovukErrorSummary("incidentStatuses")
-      }) }}
+      <div class="govuk-grid-row">
+        {{ govukCheckboxes({
```

Comment:
if you have a `div.govuk-grid-row` you need to nest a `div.govuk-grid-column-?` to make the alignment work as intended